.coafile

```diff
@@ -40,7 +40,7 @@ ignore_length_regex = ^.*https?://
 [DOCS]
 bears = SpaceConsistencyBear, LineLengthBear
 
-files = docs/**/*.rst, CONTRIBUTING.rst, README.rst
+files = **.rst
```

Comment:
This change makes bears not to check anything in docs/**/*.rst and similar in all cases.